openquake/hazardlib/site.py

```diff
@@ -205,7 +205,7 @@ def from_usgs_shakemap(cls, shakemap_array):
         arr['lat'] = shakemap_array['lat']
         arr['depth'] = numpy.zeros(n)
         arr['vs30'] = shakemap_array['vs30']
-        arr.flags.writeable = False
+        arr.flags.writeable = True
```

Comment:
This is a dramatic change, let's discuss how you can avoid that.

Reply:
> The tests are red

yes, that is weird - running that tests locally multiple times I get about 50% pass 50% fail - I edited the test, lets see, locally it passes now every time

Reply:
hmmm, I wasn't aware that it is a major change. I will investigate, something all of a sudden didn't work if I had that flag on false -  let me see what it was.

Reply:
I can put the flag back to False and the code works like before.

The function ```from_usgs_shakemap``` is only used when running one of the tests which tests```get_sitecol_shapefile``` but does not provide assets/sites. 

If (by changing the logic for debugging) I try to run the whole calculation using this function, I get the error ```ValueError: assignment destination is read-only```. But since its not possible to start a calculation without providing an exposure/hazard-curves file, this function will never be called outside of tests and debugging.

Reply:
ah and btw. - that is not something that I introduced but something that already existed before!